src/python/pants/backend/go/go_sources/parse_vendor_modules/semver.go

```diff
@@ -0,0 +1,403 @@
+// Copyright 2018 The Go Authors. All rights reserved.
```

Comment:
This file was copied mostly verbatim (except for package name) from https://github.com/golang/go/blob/master/src/cmd/vendor/golang.org/x/mod/semver/semver.go.

Reply:
I've deleted the unused bits.